app/src/main/java/fnsb/macstransit/Activities/ActivityListeners/Helpers.java

```diff
@@ -0,0 +1,339 @@
+package fnsb.macstransit.Activities.ActivityListeners;
+
+import android.util.Log;
+import android.widget.CheckBox;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.Circle;
+import com.google.android.gms.maps.model.Marker;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.android.gms.maps.model.PolylineOptions;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Locale;
+
+import fnsb.macstransit.RouteMatch.BasicStop;
+import fnsb.macstransit.RouteMatch.Route;
+import fnsb.macstransit.RouteMatch.Stop;
+
+/**
+ * Created by Spud on 2019-11-01 for the project: MACS Transit.
+ * <p>
+ * For the license, view the file titled LICENSE at the root of the project.
+ * <p>
+ * <p>
+ * This class is full of static methods that are simply used as helper methods,
+ * and thus can be used anywhere.
+ *
+ * @version 1.1
+ * @since Beta 7
+ */
+public class Helpers {
```

Comment:
Before release this class should be removed.

Consider it deprecated as of now.